applications/rubintv/values.yaml

```diff
@@ -0,0 +1,182 @@
+# Default values for rubintv.
+
+# -- Override the base name for resources
+nameOverride: ""
+
+# -- Override the full name for resources (includes the release name)
+fullnameOverride: ""
+
+# -- A special tag for letting the scripts know where they are
+# running.  Must be overridden at each site
+siteTag: ""
+
+frontend:
+  # -- If set to true, enable more verbose logging.
+  debug: false
+  # -- Prefix for rubintv's frontend API routes.
+  pathPrefix: "/rubintv"
+  # -- Settings for rubintv OCI image
+  image:
+    # -- rubintv frontend image to use
+    repository: "ghcr.io/lsst-sqre/rubintv"
+    # -- Pull policy for the rubintv image
+    pullPolicy: "IfNotPresent"
+    # -- Tag of rubintv image to use
+    # @default -- The appVersion of the chart
+    tag: ""
+  # -- Resource limits and requests for the rubintv frontend pod
+  resources: {}
+  # -- Annotations for the rubintv frontend pod
+  podAnnotations: {}
+  # -- Node selector rules for the rubintv frontend pod
+  nodeSelector: {}
+  # -- Tolerations for the rubintv frontend pod
+  tolerations: []
+  # -- Affinity rules for the rubintv frontend pod
+  affinity: {}
+
+workers:
+  # -- how many replicas to use
+  replicas: 5
+  # -- If set to true, enable more verbose logging.
+  debug: false
+  # -- Prefix for the (internal) worker API routes
+  pathPrefix: "/"
+  # -- UID to run as (site-dependent, because of filesystem perms)
+  uid: 1000
+  # -- GID to run as
+  gid: 1000
```

Comment:
I would set these to null and check in the template whether they're set and throw an error if they're not, similar to how global.host is handled in the ingress. (Also these values are being used for tucson-teststand; surely that's not correct?)